README.md

```diff
@@ -107,6 +107,10 @@ Supported crates such as `cargo-binstall` itself can also be updated with `cargo
   - You can find a full description of errors including exit codes here: <https://docs.rs/binstalk/latest/binstalk/errors/enum.BinstallError.html>
 - Can I use it in CI?
   - Yes! For GitHub Actions, we recommend the excellent [taiki-e/install-action](https://github.com/marketplace/actions/install-development-tools), which has explicit support for selected tools and uses `cargo-binstall` for everything else.
+  - Additionally, we also provide a minimalistic GitHub Action that installs `cargo-binstall`:
```

Comment:
Remove either "additionally" or "also" here, and s/minimalistic/minimal/

Reply:
thanks! good to know